RCTMapboxGL.ios.js

```diff
@@ -4,6 +4,9 @@ var React = require('react-native');
 var { NativeModules, requireNativeComponent } = React;
 
 var MapMixins = {
+  _findHandle(mapRef) {
+    return React.findNodeHandle(this.refs[mapRef]);
+  },
   setDirectionAnimated(mapRef, heading) {
```

Comment:
I see a lot of duplicated code in the API methods.  I think each method should use the helper.